src/GafferModule/PathBinding.cpp

```diff
@@ -588,6 +588,13 @@ void GafferModule::bindPath()
 				arg( "includeSequences" ) = false
 			) )
 		)
+		.def(
+			init<const std::filesystem::path &, PathFilterPtr, bool>( (
+				arg( "path" ),
+				arg( "filter" ) = object(),
+				arg( "includeSequences" ) = false
+			) )
+		)
```

Comment:
Boost evaluates function overloads in reverse order of registration, and takes the first one that it can call with the arguments it has been given. Because `str` can be coerced to `filesystem::path` in our bindings, this means that this overload is taking precedence over the original string version above. We need to move this above L584 to avoid that.